.github/renovate.json

```diff
@@ -18,17 +18,19 @@
       "matchManagers": ["npm"],
       "matchUpdateTypes": ["minor", "patch"]
     },
+    {
+      "matchManagers": ["cargo"],
+      "rangeStrategy": "widen"
+    },
```

Comment:
this would widen through breaking versions too, right?

I think that's what we want, but it does mean we likely have more manual updates to these PRs